pysper/parser/cases.py

```diff
@@ -128,29 +128,30 @@ def solr_rules():
             update(
                 event_product="solr",
                 event_category="filter_cache",
-                event_type="eviction_items_duration",
+                event_type="eviction_duration",
```

Comment:
this is the essence of the fix: produce same `eviction_duration` event for both log line formats